sdk/burdock/reader/sql/databricks.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+from pyspark.sql import Row
+from pyspark.sql.types import BooleanType
+from burdock.sql import CollectionMetadata
+from burdock.sql import PrivateReader
+from burdock.mechanisms.gaussian import Gaussian
+
+class DatabricksSparkReader:
+    def __init__(self, sparkSession, meta_path):
+        self.spark = sparkSession
+        self.schema = CollectionMetadata.from_file(meta_path)
+        self.private_reader = PrivateReader(None, self.schema, 1.0)
+
+    """
+        Executes a raw SQL string against the Spark Hive database and returns spark dataframe.
+    """
+    def execute(self, query):
+        if not isinstance(query, str):
+            raise ValueError("Please pass strings to execute.  To execute ASTs, use execute_typed.")        
+        
+        max_contrib = self.private_reader.max_contrib
+        epsilon = self.private_reader.epsilon
+        interval_widths = self.private_reader.interval_widths
+
+        subquery, fullquery = self.private_reader.rewrite(query)
+        syms = subquery.all_symbols()
+        numeric_cols = subquery.numeric_symbols()
+
+        if numeric_cols is None:
+          pass
+
+        subquery_df = self.spark.sql(str(subquery))

```

Comment:
(1) subquery
I need some help to change subquery for databricks spark.
a. remove "per_key_randome.row_num <= 5", I think it's a bug as we observed, if users' data have 6 months of data, and groupby date, will be restricted to return at most 5 months of data.
b. from "Random" to "Rand"

Reply:
@joshua-oss  is working on making tau thresholding optional which should help with 1.

We can help guide you through adding a custom serializer that resolves the Random -> Rand issue

Reply:
@joshua-oss had the PR sent, will take a look, I need incorporate his change for sure, in next iteration PR :)